src/Symfony/Bundle/FrameworkBundle/Command/SecretsListCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Exception\EncryptionKeyNotFoundException;
+use Symfony\Bundle\FrameworkBundle\Secret\SecretStorageInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Tobias Schultze <http://tobion.de>
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+final class SecretsListCommand extends Command
+{
+    protected static $defaultName = 'secrets:list';
+
+    private $secretStorage;
+
+    public function __construct(SecretStorageInterface $secretStorage)
+    {
+        $this->secretStorage = $secretStorage;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Lists all secrets.')
+            ->addOption('reveal', 'r', InputOption::VALUE_NONE, 'Display plain text value along side names');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $reveal = $input->getOption('reveal');
+        $io = new SymfonyStyle($input, $output);
```

Comment:
what about initializing it in the `Command::initialize` ?
same for other commands, as you seems to alsways use this style

Reply:
Keep it for consistency with other commands in the bundle